### ISSUE TITLE
Player movement on minimap is jerky 

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#20561]
![anim](https://user-images.githubusercontent.com/3536496/107896594-915b9400-6eeb-11eb-836b-ba7fc6aa4129.gif)

**Comment by SlayksWood** [comment]
Another player icon

**Comment by gzatravkin** [comment]
@ptitducanh check MinimapObjectIcon.cs LateUpdate, Lerp function need some improovments. Also maybe some angle problem

**Comment by SlayksWood** [comment]
Still have some problems. When I tested with 2 client, the position of my 2nd client was wrong until the avatar was stoped. After that, the position quickly moved to the desired point.
But when I asked Khairi to help make video I get that his position is not update while I am staying in one position.
Khairi has ping about 230 ms.
https://drive.google.com/file/d/113wcViUa2wEi4rMZiPni9DDO0hKG4A8G/view?usp=sharing

**Comment by SlayksWood** [comment]
```
NullReferenceException: Object reference not set to an instance of an object.
  at Unity.Entities.ComponentDependencyManager.AssertCompleteSyncPoint () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.EntityDataAccess.CheckMoreThan1024Components (System.Int32 count) [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.EntityManagerDiffer.get_EntityGuidQueryDesc () [0x00000] in <00000000000000000000000000000000>:0 
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/6121663/Player.log)

**Comment by SlayksWood** [comment]
not completely fixed. closed this one and opened a new issue #21361
